tests/manager_tests/test_links.py

```diff
@@ -203,6 +218,6 @@ def test_links_filters(self):
 
         args = QUERY_ARGS.copy()
         del args['show_all']
-        view = NoQueryListView(QUERY, SORTABLE, args, LIST_ARGS, args, URI, TOTAL_ITEMS)
+        view = NoQueryListView(QUERY, SORTABLE, DEFAULT_SORT, args, LIST_ARGS, args, URI, TOTAL_ITEMS)
```

Comment:
Is new function tested here? I don't see any new asserts. Shouldn't we also add test somewhere to e.g. `test_vulnerabilities_handler.py` to test multiple sorting columns setting?